apps/frontend/src/pages/contributors/libs/components/contributor-update-form/contributor-update-form.tsx

```diff
@@ -17,32 +22,53 @@ const ContributorUpdateForm = ({
 	contributor,
 	onSubmit,
 }: Properties): JSX.Element => {
-	const { gitEmails, name, projects } = contributor;
+	const { gitEmails, hiddenAt, name, projects } = contributor;
+
+	const isHidden = Boolean(hiddenAt);
 
 	const gitEmailsString = gitEmails.map((email) => email.email).join(", ");
 	const projectsString = projects.map((project) => project.name).join(", ");
 
-	const { control, errors, handleSubmit } = useAppForm<{
+	const { control, errors, handleSubmit, handleValueSet } = useAppForm<{
 		gitEmails: string;
+		hiddenAt: null | string;
+		isHidden: boolean;
 		name: string;
 		projects: string;
 	}>({
 		defaultValues: {
 			gitEmails: gitEmailsString,
+			hiddenAt,
+			isHidden,
 			name,
 			projects: projectsString,
 		},
 		validationSchema: contributorPatchValidationSchema,
 	});
 
+	const isHiddenValue = useFormWatch({
+		control,
+		name: "isHidden",
+	});
+
+	useEffect(() => {
+		handleValueSet("isHidden", isHidden);
+	}, [isHidden, handleValueSet]);
+
 	const handleFormSubmit = useCallback(
 		(event_: React.BaseSyntheticEvent): void => {
-			void handleSubmit((formData: { name: string }) => {
-				const payload: ContributorPatchRequestDto = { name: formData.name };
+			void handleSubmit((formData: { isHidden: boolean; name: string }) => {
```

Comment:
Why don't we just destucture it
```suggestion
			void handleSubmit(({ isHidden: boolean; name: string }) => {
```
BTW, no matter you annotate `formData` with `{ isHidden: boolean; name: string }` type, it still contains `gitEmails`, `name` and `projects`

Reply:
yep, but if I'm not mistaken `void handleSubmit(({ isHidden, name }) => {`

Reply:
agree with this option, void handleSubmit(({ isHidden, name }) => {, from the code it looks like formData should already have an appropriate type, so we don't need to define it again